### PR TITLE
DOC: mention `take_along_axis` in `choose`

### PR DESCRIPTION
**Pull request by colinsnyder**
Addresses #14117 

Mentioned `take_along_axis` in the "See Also" section of `choose`. I believe I understood the issue and added the correct documentation, but I appreciate any feedback. Thanks.

### CODE REVIEW

**Comment by colinsnyder** [comment]
Hmm. I'm not sure why it claims I have changed lines much further down in the file. I'll try to fix that.

**Comment by seberg** [comment]
You can try to fix it, since it is not super nice to have these whitespace only changes, but it is also not a real issue. A lot of editors will (correctly) automatically delete whitespace at the end, so if its not you, its likely that someone else will run into it soon. If you want to be very clean, you could make an `STY` commit, just to remove the whitespace.

**Comment by colinsnyder** [comment]
I may be incorrect, but it actually looks like I somehow removed whitespace at the end of lines, not added? Strange, but as per your comment, I suppose its better that way? Thanks.

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -380,6 +380,9 @@ def choose(a, choices, out=None, mode='raise'):
     See Also
     --------
     ndarray.choose : equivalent method
+    numpy.take_along_axis : Take values from the input array by matching 1d index
+                            and data slices. May be preferable in the case that 'choices'
```
Comment (charris) [inline numpy/core/fromnumeric.py]: Lines are too long, try to keep them <= 75 chars for documentation. Would be good to shorten the description to one line if possible, a link to the function will be generated and interested folks can click through.

File: numpy/core/fromnumeric.py [hunk 2]
```diff
@@ -380,6 +380,7 @@ def choose(a, choices, out=None, mode='raise'):
     See Also
     --------
     ndarray.choose : equivalent method
+    numpy.take_along_axis : May be preferable if 'choices' is an array
```
Comment (seberg) [inline numpy/core/fromnumeric.py]: I think choices we usually use backticks as in ``` `choices` ```. I wonder if we should say Usually preferable, or just preferable (although there are probably cases where `choose` is faster, even if just for small arrays due to much larger overhead.

**Comment by mattip** [comment]
Closes #14117 since we will not be changing the behaviour of `numpy.choose`.

The [rendered docs](https://8690-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/generated/numpy.choose.html?highlight=numpy%20choose#numpy.choose) look good, and removing the trailing whitespace is harmless. Merging.

Thanks @colinsnyder

**Pull request merged**